index.html

```diff
@@ -3474,7 +3493,7 @@ <h2>A project for first-time contributions</h2>
             <p class="name">Name</p>
             <p class="contact">
                 <i class="fa fa-twitter"></i>
-                <a href="YOUR twitter account link" target="_blank">@yourHandle</a>
+                <a href="YOUR twitter account link" target="_blank">@Artur-at-work</a>
```

Comment:
This is the template card, and it should not be changed at all. You add your Twitter account only in your card. You will need to undo this so I can merge your card. Thanks.

Reply:
Thank you for pointing that out. I have edited template to original, and pushed new commit.
Sorry for trouble.

Reply:
no trouble at all. It's all part of the normal process. Your card has been merged 👍